readme.txt

```diff
@@ -85,6 +85,35 @@ Release and roadmap notes available on the [WooCommerce Developers Blog](https:/
 
 == Changelog ==
 
+= 4.9.0 - 2021-04-12 =
+#### Enhancements
+
+- Added compatibility with the Google Analytics Integration. Block events, including cart and checkout, can now be tracked.
+
+##### Dev note
+
+Blocks are now compatible with the Google Analytics Integration: https://woocommerce.com/products/woocommerce-google-analytics/ If using Google Analytics with GTAG support (and a `G-` prefixed site ID), block events will also be tracked. This includes:
+
+- Product searches in the Product Search Block
+- Product views in the product grid blocks and All Products Block
+- Add to cart events
+- Cart item changes
+- Checkout progress events. ([4020](https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/4020))
+
+#### Bug Fixes
+
+- Use font color in payment methods border. ([4051](https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/4051))
+- Load translation file for JS files that has translatable strings. ([4050](https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/4050))
+- Stop shipping package titles line-breaks occurring in the middle of a word. ([4049](https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/4049))
+- Fixed styling issues on the cart and checkout page in TwentyX themes. ([4046](https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/4046))
+- Fix headline alignment in the empty state of the cart block. ([4044](https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/4044))
+- Fix button alignment in Featured Product and Featured Category blocks. ([4028](https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/4028))
+
+#### technical debt
```

Comment:
I realize the changelog generator did this but we should probably sentence case this to be more consistent with other headings?

Reply:
Fixed in 0cc8945631a1954b048caeaa12a8c1c730bd6951 as well.